tests/snapshots/files__unstructured-optimize.snap

```diff
@@ -3,7 +3,7 @@ source: tests/files.rs
 expression: visualization.result
 ---
 @main(v0: int) {
-  v3: bool = const true;
+  v3: bool = const false;
```

Comment:
This change seems a bit odd, but it was originally false: https://github.com/egraphs-good/eggcc/blob/86d79ea803034425ca1f81c5167950297083e6d1/tests/snapshots/files__unstructured-optimize.snap

Reply:
🤔 I've seen that change before, it probably doesn't matter for some reason